users/README.md

```diff
@@ -16,6 +16,7 @@
   'linkedin_id': string,
   'twitter_id': string,
   'instagram_id': string,
+  'skills': [],
```

Comment:
What is the size limit of a document in firestore? What happens if we add too many skills? How will we search using skills in future if we add everything to an array?

Say, I want to find out "Show all users having skill 'Node-Level-Black', what will the query look like?

Reply:
1. I didn't really keep the limit in mind...
2. we can fetch users and check which user/s have that skill in their skills prop